src/CategoryCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardHeader, IconButton, Avatar } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(1),
+    maxWidth: 500,
+    margin: '10px auto',
+  },
+}));
+
+type Props = {
+  title: string;
+  description: string;
+  avatarSrc: string;
+};
+
+export const CategoryCard: React.FC<Props> = ({
+  title,
+  description,
+  avatarSrc,
```

Comment:
Don't we need props for background color of icon? Each card has own color, that can be added to json file with remaining information.

Reply:
Yup, we need it. I've added it ✌️